src/Alpha_complex/include/gudhi/Alpha_complex.h

```diff
@@ -354,8 +355,10 @@ class Alpha_complex {
    * It also computes the filtration values accordingly to the \ref createcomplexalgorithm if default_filtration_value
    * is not set.
    *
+   * \tparam square_root_filtrations If false (default value), it assigns to each simplex a filtration value equal to
```

Comment:
(ignore the exact names I use here)
I wonder what is more natural between do_sqrt=true/false and output_squared_values=true/false. As the person writing the code, it is natural to ask if you should call sqrt. As the user... I am not sure, it might be the other one.

Reply:
Renamed as proposed on 5d9a56463